diff --git a/enzyme/include/enzyme/tuple b/enzyme/include/enzyme/tuple
index a832124..ba2c671 100644
--- a/enzyme/include/enzyme/tuple
+++ b/enzyme/include/enzyme/tuple
@@ -116,13 +116,13 @@ struct concat_with_fwd_tuple<std::index_sequence<fwd_indices...>, std::index_seq
 
 template <typename Tuple>
 __attribute__((always_inline))
-static constexpr auto tuple_cat(Tuple&& ret) {
+inline constexpr auto tuple_cat(Tuple&& ret) {
   return make_tuple_from_fwd_tuple< iseq< Tuple > >::f(impl::forward< Tuple >(ret));
 }
 
 template <typename FWD_TUPLE, typename first, typename... rest>
 __attribute__((always_inline))
-static constexpr auto tuple_cat(FWD_TUPLE&& fwd, first&& t, rest&&... ts) {
+inline constexpr auto tuple_cat(FWD_TUPLE&& fwd, first&& t, rest&&... ts) {
   return tuple_cat(concat_with_fwd_tuple< iseq<FWD_TUPLE>, iseq<first> >::f(impl::forward<FWD_TUPLE>(fwd), impl::forward<first>(t)), impl::forward<rest>(ts)...);
 }
 
